lockfile/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "phylum_lockfile"
+description = "Package lock file parsers used by Phylum"
+version = "0.1.0"
+authors = ["Phylum, Inc. <engineering@phylum.io>"]
+edition = "2021"
+rust-version = "1.60"
```

Comment:
Just for future reference: We should probably add a build with MSRV to our CI. Right now there is zero guarantee that this crate will actually build on 1.60.

Reply:
I also noticed clippy is not being run against test code.